docs/_sass/_elements.scss

```diff
@@ -29,15 +29,23 @@
 	font-weight: bold;
 	letter-spacing: .1rem;
 	font-size: 2.4rem;
+	font-family: $sless-font-secondary;
 }
-.jumbotron-sub p {
-	text-align: center;
-	color: white;
+.jumbotron-sub {
+	h2 {
+		text-align: center;
+		color: #fff;
+		color: #fff;
+    margin-top: .4rem;
+	}
+	p {
+		text-align: center;
+		color: white;
+	}
 }
 .jumbotron-sub-description {
 	width: 53%;
-    margin: 0 auto;
-    font-weight: 100;
+  margin: 0 auto;
```

Comment:
indent seems off here

Reply:
It's indented in a bit because of the sass nesting... also I'm using tabs (eek! will switch)
It looks a bit more normal in IDE:
![screenshot 2018-08-09 11 59 31](https://user-images.githubusercontent.com/123787/43910844-e7efa216-9bcb-11e8-80b6-28df65303d50.png)